configurations/create_kenya_pool.py

```diff
@@ -0,0 +1,219 @@
+                    FlowResultConfiguration("gpsdd_kilifi_s01_demog", "disabled", "disabled"),
+                ],
+                uuid_filter=rapid_pro_uuid_filter
+            )
+        ),
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/gpsdd-kiambu-textit-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                flow_result_configurations=[
+                    FlowResultConfiguration("gpsdd_kiambu_s01_demog", "constituency", "location"),
+                    FlowResultConfiguration("gpsdd_kiambu_s01_demog", "gender", "gender"),
+                    FlowResultConfiguration("gpsdd_kiambu_s01_demog", "age", "age"),
+                    FlowResultConfiguration("gpsdd_kiambu_s01_demog", "disabled", "disabled"),
+                ],
+                uuid_filter=rapid_pro_uuid_filter
+            )
+        ),
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/gpsdd-bungoma-textit-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                flow_result_configurations=[
+                    FlowResultConfiguration("gpsdd_bungoma_s01_demog", "constituency", "location"),
+                    FlowResultConfiguration("gpsdd_bungoma_s01_demog", "gender", "gender"),
+                    FlowResultConfiguration("gpsdd_bungoma_s01_demog", "age", "age"),
+                    FlowResultConfiguration("gpsdd_bungoma_s01_demog", "disabled", "disabled"),
+                ],
+                uuid_filter=rapid_pro_uuid_filter
+            )
+        ),
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/ke-vax-textit-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                flow_result_configurations=[
+                    FlowResultConfiguration("ke_constitution_review_poll_demog", "constituency", "location"),
+                    FlowResultConfiguration("ke_constitution_review_poll_demog", "gender", "gender"),
+                    FlowResultConfiguration("ke_constitution_review_poll_demog", "age", "age"),
+                    FlowResultConfiguration("ke_constitution_review_poll_demog", "disabled", "disabled"),
+                ],
+                uuid_filter=rapid_pro_uuid_filter
+            )
+        )
+    ],
+    coda_sync=CodaConfiguration(
+        coda=CodaClientConfiguration(credentials_file_url="gs://avf-credentials/coda-production.json"),
+        sync_config=CodaSyncConfiguration(
+            dataset_configurations=[
+                CodaDatasetConfiguration(
+                    coda_dataset_id="Kenya_Pool_location", #TODO Rename as Pool_Kenya_location
```

Comment:
I'm actually not sure we do need to rename this. While we're using "Pool-" in the repos for consistency and to make them easy to identify, I think the rest of the org is happy calling this the "Kenya Pool"